learning_wavelets/evaluation_scripts/exact_recon_unet_evaluate.py

```diff
@@ -0,0 +1,66 @@
+import tensorflow as tf
+from tqdm import tqdm
+
+from learning_wavelets.config import CHECKPOINTS_DIR
+from learning_wavelets.data.datasets import im_dataset_bsd68
+from learning_wavelets.evaluate import METRIC_FUNCS, Metrics 
+from learning_wavelets.models.exact_recon_unet import ExactReconUnet
+
+
```

Comment:
```suggestion
tf.random.set_seed(1)

```

You could seed the results here just like in training, I think it's a good idea